Documentation/kubernetes-on-aws.md

```diff
@@ -68,210 +81,21 @@ aws_access_key_id = AKID1234567890
-
- `The connection to the server <MASTER>:443 was refused - did you specify the right host or port?`
-
-### Export the cloudformation stack
-
-```sh
-$ kube-aws up --export
-```
-
-### Destroy
-
-When you are done with your cluster, simply run `kube-aws destroy` and all cluster components will be destroyed.
-If you created any Kubernetes Services of type `LoadBalancer`, you must delete these first, as the CloudFormation cannot be fully destroyed if any externally-managed resources still exist.
-
-### Certificates and Keys
-
-`kube-aws render` begins by initializing the TLS infrastructure needed to securely operate Kubernetes. If you have your own key/certificate management system, you can overwrite the generated TLS assets after `kube-aws render`.
-
-When `kube-aws up` creates the cluster stack, it will use whatever TLS assets it finds in the `credentials` folder at the time.
-
-This includes the certificate authority, signed server certificates for the Kubernetes API server and workers, and a signed client certificate for administrative use.
-
-* **APIServerCert, APIServerKey**
-
-	The API server certificate will be valid for the value of `externalDNSName`, as well as a the DNS names used to route Kubernetes API requests inside the cluster.
-
-	`kube-aws` does *not* manage a DNS zone for the cluster.
-	This means that the deployer is responsible for ensuring the routability of the external DNS name to the public IP of the master node instance.
-
-	The certificate and key granted to the kube-apiserver.
-	This certificate will be presented to external clients of the Kubernetes cluster, so it should be valid for external DNS names, if necessary.
-
-	Additionally, the certificate must have the following Subject Alternative Names (SANs).
-	These IPs and DNS names are used within the cluster to route from applications to the Kubernetes API:
-
-		- 10.0.0.50
-		- 10.3.0.1
-		- kubernetes
-		- kubernetes.default
-		- kubernetes.default.svc
-		- kubernetes.default.svc.cluster.local
-
-* **WorkerCert, WorkerKey**
-
-	The certificate and key granted to the kubelets on worker instances.
-	The certificate is shared across all workers, so it must be valid for all worker hostnames.
-	This is achievable with the Subject Alternative Name (SAN) `*.*.cluster.internal`, or `*.ec2.internal` if using the us-east-1 AWS region.
-
-* **CACert**
-
-	The certificate authority's TLS certificate is used to sign other certificates in the cluster.
-
-	These assets are stored unencrypted in your `credentials` folder, but are encyrpted using Amazon KMS before being embedded in the CloudFormation template.
+<div class="co-m-docs-next-step">
+  <p><strong>Did you download kube-aws?</strong></p>
+  <p><strong>Did your credentials work?</strong> We will use the AWS CLI in the next step.</p>
+  <a href="kubernetes-on-aws-render.html" class="btn btn-primary btn-icon-right"  data-category="Docs Next" data-event="Kubernetes: AWS Render">Yes, ready to configure my cluster options</a>
```

Comment:
Would be neat if the rewriter would catch this so that you could link to `*.md` here and have this link work on github _and_ coreos.com. (no priority)